cmd/runtimetest/main.go

```diff
@@ -459,6 +459,122 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func isParent(parent, child string) bool {
+	if parent == child {
+		return false
+	}
+
+	cparts := filepath.SplitList(child)
+	for i, part := range filepath.SplitList(parent) {
```

Comment:
> I think SplitList is not used for that.

You're quite right. I've fixed it to use `filepath.ToSlash` and then `strings.Split`.

> So, I think we should make exception for Windows.

Sure. I've added it using `runtime.GOOS` -- is that fine?